src/app/members/[id]/profile/page.tsx

```diff
@@ -12,7 +13,11 @@ export default function ProfilePage({
     params,
 }: {
     params: Promise<{ id: Id }>;
-}) {
+}) {    
     const userId = use(params).id;
-    return <MemberProfileContainer userId={userId} />;
+    return (
+        <PageContainer>
+            <MemberProfileContainer userId={userId} />
```

Comment:
This still isn't left justified like the other pages are. You might have something that's forcing the card/container to be centered.

Reply:
I _think_ I fixed this here 80d1f2f359e20902570496e3bc0734807f793b2b

Reply:
I can confirm it's fixed, thank you!